### PR TITLE
[Sprint Maintain] Fix UI Cart

### PR DESCRIPTION
**Pull request by Tuanlvt**
[Sprint Maintain] Fix UI Cart
Link redmine :https://dev.framgia.com/redmine/issues/123183
Effect : N/A

### CODE REVIEW

File: app/src/main/res/layout/fragment_shoppingcart.xml [hunk 1]
```diff
@@ -10,89 +11,106 @@
...
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:fontFamily="@string/font_serif"
+                    android:text="@string/title_total"
+                    android:textColor="@color/color_gray"
+                    android:textSize="@dimen/sp_16"
+                    tools:ignore="UnusedAttribute"
+                    />
+                <TextView
+                    android:id="@+id/textView4"
```
Comment (daolq3012) [inline app/src/main/res/layout/fragment_shoppingcart.xml]: pls remove this line

File: app/src/main/res/layout/fragment_shoppingcart.xml [hunk 2]
```diff
@@ -10,89 +11,106 @@
...
+                    android:gravity="center_horizontal"
+                    android:text="@{viewModel.totalPrice}"
+                    android:textColor="@color/color_black"
+                    android:textSize="@dimen/sp_20"
+                    android:textStyle="bold"
+                    />
+
+            </LinearLayout>
+            <LinearLayout
+                android:id="@+id/action_order_all"
```
Comment (daolq3012) [inline app/src/main/res/layout/fragment_shoppingcart.xml]: pls remove this line

File: app/src/main/res/layout/item_header_cart.xml [hunk 3]
```diff
@@ -14,109 +15,130 @@
...
+                    />
+                <View
+                    android:layout_width="match_parent"
+                    android:layout_height="@dimen/dp_1"
+                    android:layout_margin="@dimen/dp_2"
+                    android:background="@color/color_blue_light"
+                    />
+
+                <TextView
+                    android:id="@+id/text_price"
```
Comment (daolq3012) [inline app/src/main/res/layout/item_header_cart.xml]: pls remove this line

File: app/src/main/res/layout/item_header_cart.xml [hunk 4]
```diff
@@ -14,109 +15,130 @@
...
+                    android:textColor="@color/color_red"
+                    android:textSize="@dimen/sp_15"
+                    android:textStyle="bold"
+                    tools:ignore="UnusedAttribute"
+                    />
             </LinearLayout>
             <FrameLayout
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
+                android:id="@+id/number_order"
```
Comment (daolq3012) [inline app/src/main/res/layout/item_header_cart.xml]: pls check name convention for id of view

File: app/src/main/res/layout/fragment_shoppingcart.xml (hunk 2 above)
Comment (Tuanlvt) [inline app/src/main/res/layout/fragment_shoppingcart.xml]: It is used !

File: app/src/main/res/layout/item_header_cart.xml (hunk 3 above)
Comment (Tuanlvt) [inline app/src/main/res/layout/item_header_cart.xml]: It is used in resource

File: app/src/main/res/layout/item_content_cart.xml [hunk 5]
```diff
@@ -16,170 +16,240 @@
...
+        <RelativeLayout
             android:layout_width="match_parent"
             android:layout_height="match_parent"
-            android:orientation="horizontal"
+            android:layout_margin="@dimen/dp_5"
+            android:background="@color/color_white_light"
             >
-            <ImageView
+            <FrameLayout
                 android:id="@+id/image_product"
```
Comment (tranminhtri911) [inline app/src/main/res/layout/item_content_cart.xml]: remove this line

File: app/src/main/res/layout/fragment_shoppingcart.xml (hunk 2 above)
Comment (daolq3012) [inline app/src/main/res/layout/fragment_shoppingcart.xml]: Where has u used this id?

File: app/src/main/res/layout/item_content_cart.xml [hunk 6]
```diff
@@ -16,170 +16,240 @@
...
                             android:layout_height="wrap_content"
-                            android:layout_gravity="center"
-                            android:background="@android:color/transparent"
-                            android:onClick="@{()-> viewModel.upQuantity()}"
-                            app:srcCompat="@drawable/ic_add_black"
-                            tools:ignore="ContentDescription"
-                            />
+                            >
+                            <LinearLayout
+                                android:id="@+id/number_product"
```
Comment (daolq3012) [inline app/src/main/res/layout/item_content_cart.xml]: pls check convention of framgia

File: app/src/main/res/layout/fragment_shoppingcart.xml (hunk 2 above)
Comment (daolq3012) [inline app/src/main/res/layout/fragment_shoppingcart.xml]: Pls rename match with convention of framgia

**Pull request merged**